webapp/TargetedMS/js/QCPlotHelperBase.js

```diff
@@ -627,11 +627,13 @@ Ext4.define("LABKEY.targetedms.QCPlotHelperBase", {
 
         var mainTitle = LABKEY.targetedms.QCPlotHelperWrapper.getQCPlotTypeLabel(plotType, isCUSUMMean);
 
+        var leftMargin = 75;
+
         var basePlotConfig = this.getBasePlotConfig(id, precursorInfo.data, plotLegendData);
         var plotConfig = Ext4.apply(basePlotConfig, {
             margins : {
                 top: 65 + this.getMaxStackedAnnotations() * 12,
-                left: 75,
+                left: leftMargin + this.getYAxisLeftMarginOffset(precursorInfo),
```

Comment:
It is a rather quick calculation, but we are doing `this.getYAxisLeftMarginOffset(precursorInfo)` three times. Instead, we could just move that up to a variable at line 631 and then use it here and in the position value below.

Reply:
done